src/Commands/TerminusCommand.php

```diff
@@ -52,11 +68,21 @@ private function ensureLogin() {
             ) {
                 $auth->logInViaMachineToken(compact('email'));
             } else {
-                $message  = 'You are not logged in. Run `auth login` to ';
-                $message .= 'authenticate or `help auth login` for more info.';
+                $message  = 'You are not logged in. Run `auth:login` to ';
+                $message .= 'authenticate or `help auth:login` for more info.';
                 $this->log()->error($message);
             }
         }
         return true;
     }
+
+    /**
+     * Sets the given LoggerInterface to this class' logger property
+     *
+     * @return void
+     */
+    private function setLog()
+    {
```

Comment:
From the Symfony Console documentation, it appeared as though this process works automagically.
http://symfony.com/doc/current/components/console/logger.html
That is unless you mean that the argument should be an instance of LoggerInterface.